plugins/corepack/plugin.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "corepack",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "readme": "README.md",
   "env": {
     "PATH": "{{ .Virtenv }}/corepack-bin/:$PATH"
   },
   "shell": {
     "init_hook": [
+      "echo WARNING: corepack plugin is built into devbox since v0.10 and can be removed from devbox.json",
```

Comment:
I think it's worth mentioning that you need to set `DEVBOX_COREPACK_ENABLED` instead, rather than just removing this plugin.

Reply:
```suggestion
      "echo WARNING: corepack plugin is built into devbox since v0.10 and can be removed from devbox.json."
      "echo Set DEVBOX_COREPACK_ENABLED=true in your env instead."
```

Reply:
Yep agree, have pushed that change